harold/plugins/deploy.py

```diff
@@ -170,8 +170,9 @@ class DeployHoldListener(DeployListener):
 
     def _handle_request(self, request):
         reason = request.args['reason'][0]
-        salon_name = request.args['salon_name'][0]
-        self.monitor.hold(self.monitor.irc, None, salon_name, reason)
+        salon_name = request.args['salon'][0]
+        channel = '#' + salon_name
+        self.monitor.hold(self.monitor.irc, None, channel, reason)
```

Comment:
This method expects a slack channel name - so we prepend the `#` character since the `salon_name` will just me a slack channel without the `#` character (the name of the salon as represented in the DB).